src/main/java/com/microsoft/azure/spring/data/cosmosdb/DocumentDbFactory.java

```diff
@@ -37,10 +35,9 @@ private String getUserAgentSuffix() {
 
     public DocumentDbFactory(@NonNull DocumentDBConfig config) {
         validateConfig(config);
+        sendTelemetry();
```

Comment:
How about  we put the sendTelemetry to postconstruct instead of in constructor?

Reply:
yes，make sense, forgot that.